apps/studio/src/features/dashboard/components/CollectionTable/CollectionTable.tsx

```diff
@@ -92,7 +92,11 @@ export const CollectionTable = ({
     <Datatable
       pagination
       emptyPlaceholder={
-        <EmptyTablePlaceholder entityName="x" hasSearchTerm={false} />
+        <EmptyTablePlaceholder
+          groupLabel="collection"
+          entityName="x"
```

Comment:
The `entityName` was supposed to be updated for whatever the table is used for. Should we use entityName instead of `groupLabel`? Or vice versa, just need to remove one or the other haha

Reply:
we can't - consider the case of a folder **iwthout items**: `entityName` will be `pages` and if there's a `searchTerm` but no matching, it'll show: `no pages found`

conversely, the `groupLabel` (if this folder itself is empty) will be: `This folder is empty.` the entity name is strictly singular, for items contained within. however, the group label will be for the containing element and hence will be diff

Reply:
But entityName is currently set to `x`, which doesn't seem to make sense? Haha